README.md

```diff
@@ -49,22 +49,59 @@ Here we list formats that have been used or proposed thus far in the community (
 
 ## Standard Matrices
 
-We benchmark data formats and APIs on standard matrices. These matrices are found at the following S3 bucket.
-
-[TODO add S3 bucket details]
-
-[TODO describe matrices below]
+We benchmark data formats and APIs on standard matrices. These matrices are found in the public S3 bucket `matrix-format-test-data`.
+
+That bucket has two relevant folders, `matrices` and `sources`. The `sources` folder contains raw data from a number of different data sources, in the form that they were originally delivered. These are meant to span a range of matrix sizes and properties so we can develop tests for diverse use cases:
+
+| Source Name | Description |
+| ----------- | ----------- |
+| tenx_mouse_neuron_1M | The 1.3 million mouse neuron dataset released by 10x. |
+| tenx_mouse_neuron_20k | A sample of 20k cells from the 1.3 million cell dataset, also provided by 10x. |
+| immune_cell_census_cord_blood | Cord blood cells from the Immune Cell Atlas dataset released for the HCA preview. 384,000 cells sequenced with 10x.|
+| immune_cell_census_bone_marrow | Bone marrow cells from the Immune Cell Atlas dataset released for the HCA preview. 378,000 cells sequenced with 10x. |
+| GSE84465_whole | 3589 cells sequenced with SmartSeq2 from GEO Series GSE84465. |
+| GSE84465_split | 3589 cells sequenced with SmartSeq2 from GEO Series GSE84465, split so there is one matrix file per cell. |
+
+The `matrices` folder contains two levels of subfolders. The first is the matrix format. These are the formats that are being tested for performance among a set of different tasks.
+
+| Format Name | Description |
+| ----------- | ----------- |
+| anndata | The HDF5-based format used by [scanpy](https://github.com/theislab/scanpy) |
+| feather | [Feather format](https://github.com/wesm/feather) |
+| hdf5_10000_10000_chunks_3_compression | Dense matrix in an HDF5 dataset with (10000, 10000) chunks and gzip=3 compression |
```

Comment:
As we discussed yesterday, the "square" chunks do not make much sense for RNA-Seq data (since we don't have spatial properties for row or columns).
But I guess this is outside of this PR scope and we should address is in future.

Reply:
yea, agreed, I think this is a good start, and we can track the work discussed yesterday at #9

Reply:
Yeah let's track in a separate issue. I'll just add that some would argue that the square chunks strike a balance between accessing by gene and accessing by cell. I think we're currently assuming the former dominates, but that may not really be the case.